packages/components/src/components/hds/app-frame/index.ts

```diff
@@ -17,6 +17,7 @@ export interface HdsAppFrameSignature {
     hasMain?: boolean;
     hasModals?: boolean;
     hasSidebar?: boolean;
+    isHeaderFixed?: boolean;
```

Comment:
I was tempted to name this `hasFixedHeader`, but seeing it next to the other args, I can see how it can be confusing

Reply:
If anyone has ideas for better names, I'm fine with changing it

Reply:
although it also seems that this may go away entirely dependent upon decisions...